webpack.config.js

```diff
@@ -8,8 +8,9 @@ const ManifestPlugin = require('webpack-manifest-plugin');
 const entries = glob.sync('./vendor/**/Resources/js/index.js');
 const basePath = 'adminV2';
 
+entries.unshift('core-js/fn/promise');
```

Comment:
We basically only need this polyfill, which is used by `babelpolyfill` as well. But this way the build is a 100kb smaller. I suggest we add other stuff the same way in case we need it.